samples/csharp_dotnetcore/14.nlp-with-dispatch/README.md

```diff
@@ -107,28 +108,28 @@ Your project may be configured to rely on this secret and you should update it a
     
     Update the following line to add a prefix with the name of your bot (plus underscore '_').
     ```dotnet
-    const DISPATCH_CONFIG = '<NAME>_nlp-with-dispatchDispatch';
+    private const string DispatchKey = "<NAME>_nlp-with-dispatchDispatch";
```

Comment:
Please see the changes introduced in Microsoft#613
Are the setup steps accurate?

Reply:
Mistake 💯